tools/query_breakdown/src/main/java/com/google/bigquery/LocationTracker.java

```diff
@@ -40,6 +40,14 @@ public void addLine() {
     location.add(new ArrayList<>());
   }
 
+  /**
+   * This method inserts an element in the middle of the location ArrayList to replicate
+   * replacement
+   */
+  public void add(int x, int y, int numberToAdd) {
+    location.get(x - 1).add(y, numberToAdd);
+  }
+
```

Comment:
Seem this method, and `change()`, are only used by `replace()`. If that's the case, you should make them private.

Reply:
Got it, yeah it's definitely more of a helper function--changed!